lib/sidekiq_unique_jobs/middleware/client/strategies/testing_inline.rb

```diff
@@ -5,8 +5,12 @@ module Middleware
     module Client
       module Strategies
         class TestingInline < Unique
+          # Patch - disable TestingInline because it calls server middleware from the client side
```

Comment:
You could just delete the strategy in its entirety. It will not be used anymore.